spec/spec_helper.rb

```diff
@@ -10,7 +12,7 @@
 
 # Requires supporting ruby files with custom matchers and macros, etc,
 # in spec/support/ and its subdirectories.
-Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
+Dir[Rails.root.join('spec/support/**/*.rb')].sort.each { |f| require f }
```

Comment:
Ooh, was this you, or was this rubocop? Indeterminate load order is a pain. (And is actually fixed in one of the later versions of Ruby, just can't quite remember which one)

Reply:
It's in my commit labelled Auto-fixes so I guess it was Rubocop